### PR TITLE
Issue#280: Avoid auto population of email and password fields in create customer page

### PR DESCRIPTION
**Pull request by SunilParisi123**
Issue#280: Avoid auto population of email and password fields in create customer page

### CODE REVIEW

File: src/Web/Grand.Web.Admin/Areas/Admin/Views/Customer/_CreateOrUpdate.TabInfo.cshtml [hunk 1]
```diff
@@ -33,6 +33,9 @@
     </script>
 }
     <div class="form-horizontal">
+        @*Add dummy text and password fields to avoid autofill issue on create customer page*@
+         <input type="text" style="opacity: 0;position: absolute;">
+         <input type="password" style="opacity: 0;position: absolute;">
```
Comment (KrzysztofPajak) [inline src/Web/Grand.Web.Admin/Areas/Admin/Views/Customer/_CreateOrUpdate.TabInfo.cshtml:38]: I don't think so is a good solution

Comment (SunilParisi123) [reply src/Web/Grand.Web.Admin/Areas/Admin/Views/Customer/_CreateOrUpdate.TabInfo.cshtml:38]: @KrzysztofPajak  We tried adding Autocomplete='off' attribute in input elements but that is getting ignored by the Chrome and all major browsers.

Comment (AlexandreAndradeWIG) [reply src/Web/Grand.Web.Admin/Areas/Admin/Views/Customer/_CreateOrUpdate.TabInfo.cshtml:38]: you can pass type="password" autocomplete="new-password"

at least the password will never be auto populated

**Comment by SunilParisi123** [comment]
As it's in open state for long time and not accepred as solution closing this.

**Pull request closed without merging**